koku/api/provider/view.py

```diff
@@ -130,6 +130,9 @@ def get_queryset(self):
         by filtering against a `user` object in the request.
         """
         queryset = Provider.objects.none()
+        stat = self.request.query_params.get('stats', None)
+        if stat is not None and stat != 'false' and stat != 'true':
+            raise ProviderMethodException('the stat parameter was set to {} but it can be either \'false\' or \'true\''.format(stat))
```

Comment:
Could you provide an example error (is this being caught by the general error formatter)?

Also I would uppercase the starting 'the' and end with a period for the message.
`'The stat parameter was set to {} but it can be either \'false\' or \'true\.''.format(stat)`

I'm use to seeing the following when an error is created for 400s:
https://github.com/project-koku/koku/blob/master/koku/api/provider/serializers.py#L246